Moblin/HaishinKit/Media/VideoUnit.swift

```diff
@@ -257,6 +206,45 @@ final class VideoUnit: NSObject {
         _ = appendSampleBuffer(sampleBuffer, isFirstAfterAttach: false, applyBlur: ioVideoBlurSceneSwitch)
     }
 
+    private func startBlackImageTimer() {
```

Comment:
If possible, to slightly reduce power usage, I suggest using a single timer instead of two (gap and black). Or is there some case we need two?